src/Fragment/FragmentHandler.php

```diff
@@ -68,9 +68,11 @@ public function __construct(ContainerInterface $renderers, BaseFragmentHandler $
     /**
      * {@inheritdoc}
      */
-    public function render($uri, $renderer = 'inline', array $options = []): ?string
+    public function render($uri, $renderer = '', array $options = []): ?string
     {
-        if (!$uri instanceof FragmentReference) {
+        if (!$uri instanceof FragmentReference || '' !== $renderer) {
+            $renderer = $renderer ?: 'inline';
```

Comment:
Does this need to become `forward`, too?

Reply:
Yes, once the other PR is merged.

Reply:
It already is. 😃

Reply:
No it does not! Because we are decorating the original Symfony service, we must keep it's default arguments!